spec/template.dd

```diff
@@ -74,7 +74,18 @@ $(GNAME TemplateParameter):
         template declarations instead.)
 
 
-$(H2 $(LNAME2 explicit_tmp_instantiation, Explicit Template Instantiation))
+$(H2 $(LNAME2 template_instantiation, Template Instantiation))
+
+    $(P A template must be instantiated before use. This usually means
+    passing one or more arguments to the template which are substituted
+    into the template body, which becomes a new scoped entity.)
+
+    $(P $(RELATIVE_LINK2 function-templates, Function templates) can
+    often be implicitly instantiated by inferring the template
+    arguments from a function call. Other templates must be
```

Comment:
Avoid 'often'.
```suggestion
    be implicitly instantiated if the compiler can infer the template
    arguments from a function call. Otherwise the template must be
```